### PR TITLE
Application statuses

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- Display styled application statuses

**Related issue(s)**

### CODE REVIEW

File: core-ui/src/components/Applications/RegisterApplication/RegisterApplicationForm.js [hunk 1]
```diff
@@ -49,6 +49,9 @@ export default function RegisterApplicationForm({
   }, [formValues.name]);
 
   const applicationAlreadyExists = name => {
+    if (!checkApplicationExistQueryResult.applications) {
```
Comment (dariadomagala) [inline core-ui/src/components/Applications/RegisterApplication/RegisterApplicationForm.js]: This fix will be handled in a different PR.

File: core-ui/src/components/Applications/ApplicationList/test/ApplicationList.test.js [hunk 2]
```diff
@@ -158,8 +158,7 @@ describe('ApplicationList', () => {
         expect(
           queryByText(exampleCompassApps[i].providerName),
         ).toBeInTheDocument();
-        expect(queryByText(exampleKymaApps[i].status)).toBeInTheDocument(); //TODO: change it to some other field when we decide what is displayed on the list
       });
     });
-  });
+  }, 10000);
```
Comment (dariadomagala) [inline core-ui/src/components/Applications/ApplicationList/test/ApplicationList.test.js]: Why this change?

File: core-ui/src/components/Applications/ApplicationList/ApplicationList.js [hunk 3]
```diff
@@ -13,10 +13,23 @@ import { useQuery, useMutation } from '@apollo/react-hooks';
...
 import Badge from 'fundamental-react/Badge/Badge';
 import { useNotification } from 'react-shared';
-// import ConnectApplicationModal from '../ConnectApplicationModal/ConnectApplicationModal';
 // import LuigiClient from '@kyma-project/luigi-client';
 import ModalWithForm from '../../ModalWithForm/ModalWithForm';
 import RegisterApplicationForm from '../RegisterApplication/RegisterApplicationForm';
+import './ApplicationList.scss';
+
+const STATUSES = {
+  NOT_INSTALLED: 'not installed',
```
Comment (dariadomagala) [inline core-ui/src/components/Applications/ApplicationList/ApplicationList.js]: It's just me being picky, but if all statuses are displayed as for example `GATEWAY_NOT_CONFIGURED`, then I believe, this status should also be `NOT_INSTALLED` (with underline), not `not installed` as it doesn't match the pattern.

File: core-ui/src/components/Applications/ApplicationList/ApplicationList.js [hunk 4]
```diff
@@ -74,7 +87,7 @@ export default function ApplicationList() {
 
       if (!localAppEntry) return; // got a Kyma app that has not been registered in Compass
 
-      localAppEntry.status = kymaApp.status || 'installed'; // TODO
+      localAppEntry.status = STATUSES.INSTALLED;
```
Comment (dariadomagala) [inline core-ui/src/components/Applications/ApplicationList/ApplicationList.js]: It should be `kymaApp.status || STATUSES.INSTALLED`.
Currently, I'm seeing `installed` status on each Kyma app, instead of actual status.

File: core-ui/src/components/Applications/ApplicationList/test/ApplicationList.test.js (hunk 2 above)
Comment (Wawrzyn321) [reply core-ui/src/components/Applications/ApplicationList/test/ApplicationList.test.js]: `MockedProvider` _sometimes_ timeouts default Jest timeout of 5s, using 10s as here results in more stable tests.
